src/anbox/config.cpp

```diff
@@ -88,3 +84,17 @@ anbox::SystemConfiguration& anbox::SystemConfiguration::instance() {
   static SystemConfiguration config;
   return config;
 }
+
+anbox::SystemConfiguration::SystemConfiguration() {
+  auto gen_resource_path = [] () -> fs::path {
+    const auto snap_path = utils::get_env_value("SNAP");
+    if (!snap_path.empty()) {
+      return fs::path(snap_path) / "usr/share/anbox";
+    } else {
+      return default_resource_path;
+    }
+  };
+
+  resource_path = gen_resource_path();
+  data_path = default_path_path;
```

Comment:
default_path_path sounds wrong. Isn't this called default_data_path? If not lets change it.